WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationDetailsViewController.swift

```diff
@@ -200,6 +204,20 @@ extension NotificationDetailsViewController: UITableViewDelegate, UITableViewDat
         return cell
     }
 
+    func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        estimatedRowHeightsCache.setObject(cell.frame.height as AnyObject, forKey: indexPath as AnyObject)
+    }
+
+    func tableView(_ tableView: UITableView, estimatedHeightForRowAt indexPath: IndexPath) -> CGFloat {
+        if let height = estimatedRowHeightsCache.object(forKey: indexPath as AnyObject) as? CGFloat {
+            return height
+        }
+        return Settings.estimatedRowHeight
```

Comment:
Out of curiosity, did we have a performance issue here // any kind of visual bug? (code looks great!)